### DISCUSSION TITLE
performance for larger sites

### DISCUSSION THREAD

**Discussion by naabster** [statamic/cms#5172] [Q&A]
I am looking into statamic for a client who wants to move migrate site from Wordpress.

There are currently about 2000 blog post in Wordpress with a few more custom post types as well as tags and categories. New content is added daily. The site can sometimes get 500.000 to 1.000.000 hits / day.
I was wondering if Statamic can handle that load and still maintain dynamic parts, especially a site wide search.

I am getting mixed impressions about Statamics scalability: 
https://github.com/statamic/cms/issues/4445#issuecomment-940039014 - 5000 entries are too much for the file driver and should use database, but taxonomies are not yet ready for database driver. How does this scale up?

So my question would be: what is a ballpark limit for entries to be performant in flat file mode?

**Comment by ryanmitchell** [comment]
It depends a lot on the size of the posts, the server you are using and what type of caching you are using if any. But the figures are definitely doable and I know others here have that load and more.

You can serve Statamic in totally static mode which would definitely be able to handle that load.  

You can also put your site behind cloud flare or another caching service which would again reduce load on the site.
